cfgov/jinja2/v1/_includes/organisms/form-explainer.html

```diff
@@ -0,0 +1,198 @@
+{# ==========================================================================
```

Comment:
Copied from [the owning a home explainer template](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html). Changes include:

- removal of [tabs](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html#L23-L40)
- numbered (`category-1`) instead of named (`checklist`) categories
- handles Wagtail data structure instead of JSON
- main template code is no longer wrapped in render macro